metastats-domain/src/main/groovy/life/qbic/metastats/filter/FilterExperimentData.groovy

```diff
@@ -169,7 +158,7 @@ class FilterExperimentData implements FilterExperimentDataInput {
      * @param entryProps which contains all properties along with the filenames
      * @return validation status as boolean
      */
-    static boolean validFilenames(HashMap entryProps) {
+    boolean validFilenames(HashMap entryProps) {
```

Comment:
```suggestion
    void validateFilenames(HashMap entryProps) throws IllegalArgumentException{
```
Either rename the method and throw an error on failure or rename it to `boolean isAllFilenamesValid()` or similar. Your current method name does not communicate a clear boolean expression imo. I would suggest the approach with the exception (compare to json parser) to implement Tell, don't ask principles. 

Furthermore I would suggest to split this method into the interation across all filenames and an individual method that only validates a single filename.

Reply:
Thanks for your remark, but imo this is not part of the feature of this PR. Therefore, I would like to stick it and open (if desired) an issues for changing method names or refactoring code. Would this be ok for you?

Reply:
would be fine with me. You can then simply link the issue here.

Reply:
On the other hand this does also affect your logging location. So the location in your code where you go from failing to logging.

Reply:
#7